<launch>
  <node name="joy" pkg="joy" type="joy_node" args="_autorepeat_rate:=10"/>

  <node name="magrobot_groundstation" pkg="magrobot_groundstation" type="joy2twist_node">
			<param name="enable_button" value="1" />
			<param name="enable_turbo_button" value="0" />
			<param name="scale_angular" value="3.14" />
			<param name="scale_linear" value="0.5" />
			<param name="scale_linear_turbo" value="1" />

	</node>
  
  <node name="model_publisher" pkg="model_publisher" type="model_publisher" output="screen">
  <rosparam param="base_frame"> "base_link" </rosparam>
  <rosparam param="topic_name"> "robot_model" </rosparam>
  <rosparam param="offset_x"> 0 </rosparam>
  <rosparam param="offset_y"> 0.0  </rosparam>
  <rosparam param="offset_z"> 0.0 </rosparam>
  <rosparam param="offset_roll"> 0.0 </rosparam>
  <rosparam param="offset_pitch"> 0.0 </rosparam>
  <rosparam param="offset_yaw"> 0.0 </rosparam>
  <rosparam param="model_path"> "file:///home/cle/catkin_ws/src/magrobot/magrobot_platform/model/Robot.stl"  </rosparam></node>
</launch>
